api/src/main/java/org/openmrs/hl7/impl/HL7ServiceImpl.java

```diff
@@ -9,24 +9,22 @@
  */
 package org.openmrs.hl7.impl;
 
-import java.io.File;
```

Comment:
Could you please be sure your IDE is properly configured to support OpenMRS? This import order should be maintained.

Reply:
@ibacher thanks you very much .